docs/how-to/ccv0.sh

```diff
@@ -51,28 +51,19 @@ export CONTAINER_ID="${CONTAINER_ID:-0123456789}"
 source /etc/os-release || source /usr/lib/os-release
 grep -Eq "\<fedora\>" /etc/os-release 2> /dev/null && export USE_PODMAN=true
 
-export BATS_TEST_DIRNAME="${tests_repo_dir}/integration/containerd/confidential"
+
 # If we've already checked out the test repo then source the confidential scripts
-[ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/lib.sh"
+if [ "${KUBERNETES}" == "yes" ]; then
+    export BATS_TEST_DIRNAME="${tests_repo_dir}/integration/kubernetes/confidential"
+    [ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/lib.sh"
+else
+    export BATS_TEST_DIRNAME="${tests_repo_dir}/integration/containerd/confidential"
+    [ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/lib.sh"
+fi
 
-export RUNTIME_CONFIG_PATH=/etc/kata-containers/configuration.toml
+[ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/../../confidential/lib.sh"
 
-debug_output() {
```

Comment:
@GeorginaKin , I'd like to understand this change. Are you removing those debug functions because of my comment in https://github.com/kata-containers/kata-containers/pull/4221#discussion_r869603387 , you aren't?

They seem legit debugging function. My comment was about the `echos` that sometime we add while developing a code but that should be removed afterwards. I just wanted to make clear my comments :)

Reply:
@wainersm - it was my suggestion after I hit issues with `DEBUG` being unset. Those debug functions were some inelegant code that I added when I first wrote this script and the `set -x` that I added since gives >95% of the value, so I suggested to George that she remove it. Happy to discuss further though

Reply:
Got it. I was worried about it was due my early comment. So we are fine here :)